iOS_SDK/OneSignalSDK/Source/OSMessagingController.m

```diff
@@ -548,11 +574,10 @@ - (BOOL)isClickAvailable:(OSInAppMessage *)message withClickId:(NSString *)click
 
 - (void)sendClickRESTCall:(OSInAppMessage *)message withAction:(OSInAppMessageAction *)action {
     let clickId = action.clickId;
-    // Make sure no click tracking is performed for IAM previews
     // If the IAM clickId exists within the cached clickedClickIds return early so the click is not tracked
     // unless that click is from an IAM with redisplay
     // Handles body, button, or image clicks
-    if (message.isPreview || ![self isClickAvailable:message withClickId:clickId])
+    if (![self isClickAvailable:message withClickId:clickId])
```

Comment:
We are checking for this earlier now right? Thats why we can remove the preview check?

Reply:
thats right